src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -136,8 +138,8 @@ private enum Mode
         /// <remarks>
         /// If <c>true</c> then a query string "v" with the encoded content of the file is added.
         /// </remarks>
-        [HtmlAttributeName(FileVersionAttributeName)]
-        public bool? FileVersion { get; set; }
+        [HtmlAttributeName(AppendVersionAttributeName)]
+        public bool? AppendVersion { get; set; }
```

Comment:
@Eilon IIRC this might be historical / hysterical. Currently determine mode using `TagHelperContext` (its `AllAttributes` collection) and not the properties themselves. So nullable doesn't look mandatory anymore.

But I suggest we clean this up separately if @DamianEdwards agrees.